dwl.c

```diff
@@ -1769,7 +1790,14 @@ rendermon(struct wl_listener *listener, void *data)
 	if (!wlr_output_attach_render(m->wlr_output, NULL))
 		return;
 
-	if (render) {
+	bool needs_frame;
```

Comment:
Suckless style: declarations before statements, `int` not `bool`, one-line blocks without braces where not confusing.

Reply:
Another note here: we can probably remove `wlr_output_attach_render`, since that's the first thing that `wlr_output_damage_attach_render` calls.

Reply:
(Oh, I see about the `bool`. Ugh, thanks "unopinionated" wlroots.)

Reply:
I'm unfamiliar with the suckless coding style and actually any C coding standards for that matter. I'm happy to change this, but wlroots does use a bool here. Leaving for as is now.

Reply:
Removed the `wlr_output_attach_render` a new revision.